packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -8,6 +8,7 @@ import { Listr } from 'listr2'
 import { paramCase } from 'param-case'
 import pascalcase from 'pascalcase'
 import terminalLink from 'terminal-link'
+import { prettify } from '../../../lib/index'
```

Comment:
You're getting a linting error here; you can fix it by `yarn lint:fix` after running `yarn build`

Reply:
Thank you for the pointer! I will keep this in mind in the future ;)